doc/demuxers.texi

```diff
@@ -749,6 +749,12 @@ cast to int32 are used to adjust onward dts.
 
```

Comment:
The patch was based off master over 3 months ago. @softworkz Is there something that is not correctly happening when pull requests are created from this repository? What can we do to make sure the patch is from master?

What specifically is ad hoc about the change? We have a very good reason to export the information since extraction of that value is a business requirement on our end. 

You are correct the field is only defined in QuickTime. Is there a suggestion that you can provide that would limit this change only to export if the video is QuickTime? Would that help allow this change to be approved?